src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/LuceneLocationsExtractor.scala

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.locations
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.Location
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.client.FeatureServiceClient
+import com.microsoft.partnercatalyst.fortis.spark.transforms.topic.LuceneKeyphraseExtractor
+
+class LuceneLocationsExtractor(
+                                lookup: Map[String, Set[Location]],
+                                featureServiceClient: FeatureServiceClient,
+                                placeRecognizer: Option[PlaceRecognizer] = None,
+                                locationLimit: Int = Int.MaxValue,
+                                ngrams: Int = 3
+                              ) extends LocationsExtractor(lookup, featureServiceClient, placeRecognizer, locationLimit, ngrams) {
+
+  lazy private val keyphraseExtractor = new LuceneKeyphraseExtractor("UNKNOWN", lookup.keySet, locationLimit)
```

Comment:
Using `UNKNOWN` as the language since the LocationsExtractor doesn't have a notion of language (and I didn't want to introduce that extra change at the moment). When passing a language value for which there is no specific `Analyzer`, the `StandardAnalyzer` will be used. The `StandardAnalyzer` class will tokenize in a language-neutral way for languages that use the Latin alphabet.